src/backend/web/handlers/tests/test_webhooks.py

```diff
@@ -0,0 +1,120 @@
+from unittest.mock import patch
+
+from werkzeug.test import Client
+
+from backend.common.consts.client_type import ClientType
+from backend.common.helpers.tbans_helper import TBANSHelper
+from backend.common.models.mobile_client import MobileClient
+from backend.common.models.user import User
+
```

Comment:
Can we add a few more tests to tests cases like attempting to send a verification for a webhook that doesn't belong to a user? Seems like there's some gaps around testing for error states as well